### ISSUE TITLE
Test dataset distribution

### ISSUE THREAD

**Issue by mpatsis** [IBM/TabFormer#30]
Hi,

I am trying to replicate the results from the paper.
Is the distribution used in the test set the same as the train set before upsampling the fraud class (imbalanced) or have you used a balanced test set?
Could you also please comment on how you split the data? Did you split randomly or did you use the bottom part of the dataset as the test set?

Kind regards,
Rafael

**Comment by mpatsis** [comment]
The upsampling is performed only on the training set.
The front part is used as the train and the tail of the file as test.
